backend/apps/volontulo/serializers.py

```diff
@@ -204,7 +209,10 @@ def get_is_administrator(obj):
     def get_organizations(self, obj):
         """Returns organizations that user belongs to."""
         qs = obj.userprofile.organizations.all()
-        return OrganizationSerializer(qs, many=True, context=self.context).data
+        return OrganizationSerializer(qs, many=True, context=self.context).data\
```

Comment:
What's this `\` does on the end of line?

Moreover - it cause also Travis to fail: `./apps/volontulo/serializers.py:212:80: E501 line too long (80 > 79 characters)`